confidential-data-hub/README.md

```diff
@@ -31,14 +39,16 @@ Confidential resource providers (flag `RESOURCE_PROVIDER`)
 | kbs                 | For TDX/SNP/Azure-SNP-vTPM based on KBS Attestation Protocol       |
 | sev                 | For SEV based on efi secret pre-attestation                        |
 
-Note: `offline-fs` is built-in, we do not need to manually enable. If no `RESOURCE_PROVIDER`
-is given, all features will be enabled.
+Note:
+- `offline-fs` is built-in, we do not need to manually enable. 
+- If no `RESOURCE_PROVIDER` flag is given, then all the resource providers will be enabled by default
```

Comment:
As mentioned, we should also delete this sentence (my fault, I did not delete it before).